testsuite/MDAnalysisTests/lib/test_util.py

```diff
@@ -124,7 +126,7 @@ def test_strings(self):
         assert_equal(util.iterable(u"unicode string"), False)
 
 
-class TestFilename(object):
+class TestFilename(TestCase):
```

Comment:
No, I'm using it because on classes that inherit from `TestCase` get their `setUp` and `tearDown` methods executed.